src/LeanCode.ContractsGenerator.Tests/ExampleBased/Notifications/NotificationTagGenerators.cs

```diff
@@ -0,0 +1,118 @@
+using Xunit;
+using CompilationTagGenerator = LeanCode.ContractsGenerator.Generation.NotificationTagGenerator;
+using ContractsTagGenerator = LeanCode.Contracts.NotificationTagGenerator;
+
+namespace LeanCode.ContractsGenerator.Tests.ExampleBased.Notifications;
+
+public class NotificationTagGenerators
```

Comment:
Let's move this out of the `ExampleBased` tests - these are far from being example-based in the same manner as the other example-based tests. Let's treat these as normal unit tests.